corehq/apps/reminders/templates/reminders/partial/scheduled_reminders.html

```diff
@@ -1,8 +1,24 @@
-{% extends "reminders/reminders_base.html" %}
+{% extends "style/bootstrap3/base_section.html" %}
 {% load hq_shared_tags %}
+{% load i18n %}
 
 {% block js-inline %}{{ block.super }}
     <script>
+        $(function (){
```

Comment:
we can remove everything in this jquery block since this view doesn't have any editing capabilities